config/settings.yml

```diff
@@ -12,6 +12,13 @@ financial:
     csv: true
     xml: true
 
+converters:
+  default:
+    product_to_journal_rows: Converters::ProductToJournalRows
+    order_detail_to_journal_rows: Converters::OrderDetailToJournalRows
+  split_accounts/split_account:
+    order_detail_to_journal_rows: SplitAccounts::Converters::OrderDetailToJournalRows
```

Comment:
Another thought: I think I'm now ok with having the default converter be configurable here, but there's a part of me that feels a little weird about having the split_accounts config here. Seems like that should be part of the engine, though I'm not sure how best to handle that.